srv/components/provisioner/files/cluster_sls.template

```diff
@@ -42,7 +42,7 @@ cluster:
     rack_id:
     site_id:
     storage_set_id: {{ storage_set_id }}
-    node_id:
+    node_id: {{ salt['mine.get'](srvnode , 'get_nodes_id')[srvnode] }}
```

Comment:
@mazinamdar Are you sure that mine data would be updated at the time this template is rendered?

Reply:
yes, I have tried this on single node, node_id is getting assigned in cluster.